lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -211,14 +205,14 @@ public void testForceClaimTaskFromOtherUser()
         assertEquals(createdTask.getOwner(), "other_user");
 
         Instant beforeForceClaim = Instant.now();
-        waitAMillisecond();
         Task taskAfterClaim = taskService.forceClaim(createdTask.getId());
 
         assertEquals(CurrentUserContext.getUserid(), taskAfterClaim.getOwner());
-        assertTrue(taskAfterClaim.getModified().isAfter(beforeForceClaim));
-        assertTrue(taskAfterClaim.getClaimed().isAfter(beforeForceClaim));
+        assertTrue(isBeforeOrEqual(beforeForceClaim, taskAfterClaim.getModified()));
+        assertTrue(isBeforeOrEqual(beforeForceClaim, taskAfterClaim.getClaimed()));
+        assertTrue(isBeforeOrEqual(taskAfterClaim.getCreated(), taskAfterClaim.getModified()));
```

Comment:
They have to be equal.

Reply:
yes.

Reply:
I don't get it. Why should created equals modify? We claimed the task after creation